internal/ignition/ignition.go

```diff
@@ -1388,16 +1392,33 @@ func QuoteSshPublicKeys(sshPublicKeys string) string {
 	return strings.ReplaceAll(sshPublicKeys, "\n", `", "`)
 }
 
-func getUserSSHKey(sshKey string) string {
+func getUserSSHKey(sshKey string) (string, error) {
 	if sshKey == "" {
-		return ""
+		return "", nil
+	}
+	keys := strings.Split(sshKey, "\n")
+	validKeys := []string{}
+	// filter only valid non empty keys
```

Comment:
Please consider fix the input when receiving it from the customer

Reply:
The thing is that if we already have some values in DB like `ssh-rsa key coyote@acme.com\n\n\tssh-rsa key2 coyote@acme.com` Then we might just want to filter out the empty strings out of the keys array to make it cleaner and more readable. 
But that's true, we should also "fix" it before DB insertion.